src/PrizePool.sol

```diff
@@ -241,11 +245,19 @@ contract PrizePool {
         }
     }
 
+    function _nextDrawEndsAt() internal view returns (uint64) {
```

Comment:
Should be `uint32` now that we use it for `Observation.timestamp`.

```suggestion
    function _nextDrawEndsAt() internal view returns (uint32) {
```

Reply:
It's actually 64bit!

Check it out: 

```
uint64 internal lastCompletedlastCompletedDrawStartedAt_;
```

It's better because then we can ignore overflow.  We only needed 32 bits in the TWAB to squeeze it super tight.